containres/ideavirtualdesktopcontroller/app/app_main.py

```diff
@@ -36,6 +36,15 @@ def main(**kwargs):
         module_id = EnvironmentUtils.idea_module_id(required=True)
         module_set = EnvironmentUtils.idea_module_set(required=True)
         aws_region = EnvironmentUtils.aws_default_region(required=True)
+        aws_client_endpoints = {
+            "s3": EnvironmentUtils.s3_endpoint(required=False),
+            "dynamodb": EnvironmentUtils.dynamodb_endpoint(required=False),
+            "sqs": EnvironmentUtils.sqs_endpoint(required=False),
+            "cognito-idp": EnvironmentUtils.cognito_endpoint(required=False),
+            "kinesis": EnvironmentUtils.kinesis_endpoint(required=False),
+            "secretsmanager": EnvironmentUtils.secretsmanager_endpoint(required=False),
+        }
+        # todo add some flag here to create the DB if not exist using some config file as default values
```

Comment:
old comment this is now being handled in some init container orchestration to this svc